sync/class.jetpack-sync-listener.php

```diff
@@ -259,31 +261,29 @@ function get_actor( $current_filter, $args ) {
 			'is_wp_admin'      => is_admin(),
 		);
 
-		if ( $this->should_send_ip_address_with_actor( $current_filter ) ) {
+		if ( $this->should_send_user_data_with_actor( $current_filter ) ) {
 			require_once( JETPACK__PLUGIN_DIR . 'modules/protect/shared-functions.php' );
 			$actor['ip'] = jetpack_protect_get_ip();
+			$actor['user_agent'] = isset( $_SERVER['HTTP_USER_AGENT'] ) ? $_SERVER['HTTP_USER_AGENT'] : 'unknown';
 		}
 
 		return $actor;
 	}
 
-	function should_send_ip_address_with_actor( $current_filter ) {
-		if ( ! in_array( $current_filter, array( 'wp_login', 'wp_logout', 'jetpack_valid_failed_login_attempt' ) ) ) {
-			// Only send IP Address with login-related events
-			return false;
-		}
-
+	function should_send_user_data_with_actor( $current_filter ) {
+		$should_send = in_array( $current_filter, array( 'wp_login', 'wp_logout', 'jetpack_valid_failed_login_attempt' ) ) ?
+			true : false;
```

Comment:
We can make it even shorter. 
`$should_send = in_array( $current_filter, array( 'wp_login', 'wp_logout', 'jetpack_valid_failed_login_attempt' ) )`